mri/numerics/cost.py

```diff
@@ -13,3 +13,4 @@
 
 # Package import
 from mri.reconstruct.cost import DualGapCost
+from mri.reconstruct.cost import GenericCost
```

Comment:
I think importing the 2 on the same line is ok

Reply:
Cool, will update